src/add/operator/exhaust.ts

```diff
@@ -2,4 +2,4 @@ import {Observable} from '../../Observable';
 import {exhaust} from '../../operator/exhaust';
 Observable.prototype.exhaust = exhaust;
 
-export var _void: void;
\ No newline at end of file
+export var _void: void;
```

Comment:
I don't think we currently have strict rules for those. Personally I clean up endings if I could find it in my changes, but in this case if file hasn't changed it'd better to unstage simply.